docs/specs/tripgo.openapi.yaml

```diff
@@ -5621,6 +5626,21 @@ components:
           type: number
         unit:
           type: string
+    GenericError:
+      type: object
+      properties:
+        error:
+          type: string
+          description: "A description of the error"
+        errorCode:
+          type: integer
+          description: "Optional error code"
+        usererror:
+          type: boolean
+          description: "if this error should be shown to the user"
+      required:
+        - error
+        - usererror
```

Comment:
couldn't we just reuse the one that has the error codes? this has the same structure.

Reply:
It does, but all those routing error codes don’t make sense in the more generic errors or reporting them in the docs for endpoints that aren’t routing related.

Reply:
i see!